lib/cmock_header_parser.rb

```diff
@@ -43,12 +45,55 @@ def parse(name, source)
 
     { :includes  => nil,
       :functions => @funcs,
-      :typedefs  => @typedefs
+      :typedefs  => @typedefs,
+      :normalized_source    => transform_inline_functions(source),
```

Comment:
Maybe only do this step if we have this feature enabled? Otherwise this is a lot of overhead that is just going to be thrown away.

Reply:
Yes indeed, not very efficient like this...